controller/users.go

```diff
@@ -197,11 +217,11 @@ func ConvertUser(request *goa.RequestData, identity *account.Identity, user *acc
 		Type: workitem.SimpleType{Kind: workitem.KindString},
 	}
 
-	converted := app.Identity{
-		Data: &app.IdentityData{
+	converted := app.User{
+		Data: &app.UserData{
 			ID:   &id,
 			Type: "identities",
```

Comment:
Tracking the UI-side change here 
https://github.com/fabric8io/fabric8-ui/issues/413